### PR TITLE
JPN-540 Design QA found bugs

### PR DESCRIPTION
**Pull request by adamkarminski**
### Description
- "View more" contributors link was not displaying in some cases - fixed
- Fixed spacing and alignment in the contributors module and modal
### Reviewers

@Wikia/spitfires @perjg @d34th4ck3r

### CODE REVIEW

File: extensions/wikia/CommunityPage/styles/components/_contributors-module.scss [hunk 1]
```diff
@@ -49,7 +58,8 @@
 		span {
 			display: block;
 			font-size: 16px;
-			line-height: 18px;
+			height: 20px;
```
Comment (adamkarminski) [inline extensions/wikia/CommunityPage/styles/components/_contributors-module.scss]: Use a variable

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 2]
```diff
@@ -35,7 +35,7 @@
 	'communitypage-recently-joined' => 'Recent New Contributors',
 	'communitypage-recent-activity' => 'View all recent activity »',
 	'communitypage-all-members' => 'All Members',
-	'communitypage-view-all-members' => 'View All Contributors »',
+	'communitypage-view-all-members' => 'View all contributors »',
```
Comment (perjg) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php:38]: There are also some other capitalization issues, reported and fixed in JPN-542.

**Comment by perjg** [comment]
LGTM

File: extensions/wikia/CommunityPage/styles/components/_contributors-module.scss [hunk 3]
```diff
@@ -65,38 +76,75 @@
...
 			padding: 0 $modal-horizontal-padding;
+		}
 
-			.community-page-all-contributors-item {
-				border-bottom: 2px solid $color-page-border-subtle;
-				padding: 15px 0;
-			}
+		.community-page-contributors-list-item {
+			padding: 12px 0;
+			border-bottom: 2px solid $color-page-border-subtle;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_contributors-module.scss:132]: Properties order

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php (hunk 2 above)
Comment (adamkarminski) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php:38]: Let's take care of them as part of JPN-542 :)

**Pull request merged**